app/controllers/api/v1/errors_controller.rb

```diff
@@ -11,16 +11,16 @@ def index
   end
 
   def show
-    @grouped_issue = GroupedIssue.find(params[:id])
+    @grouped_issue = current_group
```

Comment:
asta e o prostie. Poti sa folosesti cancan sa loadeze grouped_issue prin `class: GroupedIssue` ar trebuii sa mearga asta!! ... fara sa avem nevoie de metoda `current_group` care la fel e bagata aiurea in api_controller nu are ce cauta acolo ca nu ai nevoie de ea in tot apiu ...